src/EnergyPlus/Furnaces.cc

```diff
@@ -6203,12 +6226,34 @@ namespace Furnaces {
 
                         if (state.dataFurnaces->Furnace(FurnaceNum).NumOfSpeedCooling > 0) {
                             SetOnOffMassFlowRate(state, FurnaceNum, AirLoopNum, OnOffAirFlowRatio, OpMode, QZnReq, MoistureLoad, PartLoadRatio);
-                            CalcVarSpeedHeatPump(
-                                state, FurnaceNum, false, Off, 1, 0.0, 0.0, SensibleOutput, LatentOutput, 0.0, 0.0, OnOffAirFlowRatio, SUPHEATERLOAD);
+                            CalcVarSpeedHeatPump(state,
+                                                 FurnaceNum,
+                                                 false,
+                                                 CompressorOperation::Off,
+                                                 1,
+                                                 0.0,
+                                                 0.0,
+                                                 SensibleOutput,
+                                                 LatentOutput,
+                                                 0.0,
+                                                 0.0,
+                                                 OnOffAirFlowRatio,
+                                                 SUPHEATERLOAD);
                         } else {
                             SetOnOffMassFlowRate(state, FurnaceNum, AirLoopNum, OnOffAirFlowRatio, OpMode, QZnReq, MoistureLoad, PartLoadRatio);
-                            CalcFurnaceOutput(
-                                state, FurnaceNum, false, 0, Off, 0.0, 0.0, 0.0, 0.0, SensibleOutput, LatentOutput, OnOffAirFlowRatio, false);
+                            CalcFurnaceOutput(state,
```

Comment:
Why is this function called `CalcFurnaceOutput` and not `CalcFurnace`?

Reply:
There is another function called `CalcFurnaceResidual` that calls `CalcFurnaceOutput`. Shall I rename `CalcFurnaceOutput` to `CalcFurnace`?

Reply:
Don't bother. I shouldn't have said anything.

Reply:
😬